lib/openai/client.rb

```diff
@@ -107,5 +107,17 @@ def beta(apis)
         client.add_headers("OpenAI-Beta": apis.map { |k, v| "#{k}=#{v}" }.join(";"))
       end
     end
+
+    def inspect
+      sensitive_attributes = %i[@access_token @organization_id @extra_headers]
```

Comment:
I included `extra_headers`, since my thinking was this may contain other API keys and it is better to hide too much than potentially show something sensitive. Let me know if others disagree